nexus/reconfigurator/planning/src/system.rs

```diff
@@ -1473,6 +1474,9 @@ impl Sled {
                 ),
                 // XXX: return something more reasonable here?
                 zone_image_resolver: ZoneImageResolverInventory::new_fake(),
+                // TODO-K[omicron#9517]: Have the ability to add failed health
+                // checks here
+                health_monitor: HealthMonitorInventory::new(),
```

Comment:
There are enough of these that I wonder if it's worth tracking better than a `TODO`.  Just a thought: `HealthMonitorInventory::stub()`, and then there's just one TODO on that saying "we should replace this with something real for all the callers"?

Reply:
Hmm, `HealthMonitorInventory::new()` isn't really a stub and it's mostly used in places (mostly tests) where having an empty `HealthMonitorInventory` makes sense. So, I wouldn't want to change this type. I left the breadcrumbs there because the places with the TODOs aren't super obvious and I didn't want to forget where I meant to have emulated failed health checks. I can update the issue to point where these checks will go though.

Reply:
Ah! gotcha, I've already updated the issue, and it's just a few places, so I think that should be enough thanks!